src/store/getters/state-getters.ts

```diff
@@ -491,7 +491,8 @@ export const isIncorporationApplicationValid = (state: StateIF): boolean => {
   const isBaseStepsValid = (
     getCreateShareStructureStep(state).valid &&
     getEffectiveDateTime(state).valid &&
-    getIncorporationAgreementStep(state).valid
+    getIncorporationAgreementStep(state).valid &&
+    getCourtOrderStep(state).valid
```

Comment:
The getCourtOrderStep() method was not being called to check for validation.

Reply:
You found it. Nice!

Now this is checked for all "base" entity types (BEN, BC, CC, ULC). Is this step valid for those other types? You might need a conditional here.

Reply:
The court order component is now rendered for BEN/BC/CCC/ULC, so this check should be valid for all "base" types.
= resolved